netutils/config/compliance.py

```diff
@@ -1,8 +1,10 @@
 """Filter Plugins for compliance checks."""
 
+import typing as t
+
 from . import parser  # pylint: disable=relative-beyond-top-level
 
-parser_map = {
+parser_map: t.Dict[str, t.Callable] = {  # type: ignore
```

Comment:
Type of `parser_map` should be `t.Dict[str, t.Type[BaseConfigParser]]`. `t.Type[SomeClass]` is the type hint for the actual class type or any subclasses (rather than instances of the class).